src/common/services/email/email.service.ts

```diff
@@ -64,8 +114,8 @@ export class EmailService {
         context: {
           imageUrl,
           eventName,
-          ticketUrl
-        }
+          ticketUrl,
+        },
```

Comment:
please, adjust your prettier to our previously established code style. it doesnt make sense when we for each commit add such changes